scss/_patterns_chip.scss

```diff
@@ -1,18 +1,19 @@
 @import 'settings';
 
 @mixin vf-p-chip {
-  $chip-vertical-padding: map-get($nudges, nudge--small);
-  $chip-height: map-get($line-heights, small) + 2 * $chip-vertical-padding;
-  $chip-margin-bottom: $spv-outer--small-scaleable - 2 * $chip-vertical-padding;
+  $chip-line-height: map-get($line-heights, default-text);
+  $chip-vertical-padding: 0;
```

Comment:
All uses of this variable are obsolete so you can remove it once addressed below.